torch/_inductor/select_algorithm.py

```diff
@@ -366,6 +372,46 @@ def __init__(
         self.template_out: Optional[str] = None
         self.ops_handler: Optional[V.WrapperHandler] = None  # type: ignore[name-defined]
 
+        # The generated code is not dependent on the input nodes names, or symbolic sizes names.
+        # However, some of the variables returned by generate_and_load that are computed during the
```

Comment:
Can we have something louder ? then full list

XXX: NEW FIELDS WHICH ARE MODIFIED DURING KERNEL DEFINITION (not including fusion) MUST BE ADDED HERE